src/modals/NewAddressModal.tsx

```diff
@@ -103,22 +103,30 @@ const NewAddressModal = ({ title, onClose, singleAddress }: NewAddressModalProps
 
   return (
     <CenteredModal title={title} onClose={onClose}>
-      <Section>
-        <AddressMetadataForm
-          label={addressLabel}
-          setLabel={setAddressLabel}
-          mainAddressMessage={mainAddressMessage}
-          isMain={isMainAddress}
-          setIsMain={setIsMainAddress}
-          isMainAddressToggleEnabled
-          singleAddress={singleAddress}
-        />
-        {!singleAddress && (
-          <InfoBox Icon={Info} contrast noBorders>
-            The group number will be automatically be appended to the addresses’ label.
-          </InfoBox>
-        )}
-      </Section>
+      {!passphraseHash && (
+        <Section>
+          <AddressMetadataForm
+            label={addressLabel}
+            setLabel={setAddressLabel}
+            mainAddressMessage={mainAddressMessage}
+            isMain={isMainAddress}
+            setIsMain={setIsMainAddress}
+            isMainAddressToggleEnabled
+            singleAddress={singleAddress}
+          />
+          {!singleAddress && (
+            <InfoBox Icon={Info} contrast noBorders>
+              The group number will be automatically be appended to the addresses’ label.
+            </InfoBox>
+          )}
+        </Section>
+      )}
+      {passphraseHash && singleAddress && (
+        <InfoBox contrast noBorders>
+          If you don&apos;t mind which group the address will be in, go ahead and hit that &quot;Generate&quot; button.
```

Comment:
This makes me wonder as a user why I'd mind

Reply:
Did you watch the video where I explain why I put this there? 😅  (audio on, plz!)

Reply:
I watched it with no audio, yep :V _one moment later_ Ok so the message still itself doesn't explain why they'd mind, it could be written in a way that explains what will happen if generate is pressed instead making it more clear and not make them wonder what it really means: `"By default an address of any group will be generated. This is normal behavior. If you need to generate an address of a specific group please expand the section below. "` Here the user is assured that it's normal behavior and doesn't have to think "do I really mind?" - y'know?

Reply:
OK, good suggestion, I'll rephrase, thanks a lot! <3

Reply:
How'bout this? 
<img width="616" alt="image" src="https://user-images.githubusercontent.com/1579899/177989503-4c15ffed-2103-494c-8bcc-69fd09043dc4.png">

Reply:
Yeah that's good